src/psyclone/domain/lfric/lfric_kern.py

```diff
@@ -674,8 +674,8 @@ def gen_code(self, parent):
                            entity_decls=["cell"]))
         # Import here to avoid circular dependency
         # pylint: disable=import-outside-toplevel
-        from psyclone.dynamo0p3 import DynLoop
-        parent_loop = self.ancestor(DynLoop)
+        from psyclone.domain.lfric import LFRicLoop
```

Comment:
Again, can this be moved up?

Reply:
no - LFRicLoop uses LFRicKern and LFRicKern uses LFRicLoop